test/13_loki-backend.ts

```diff
@@ -111,7 +111,7 @@ describe('loki backend', () => {
 		const loki = new LokiBackend();
 		await new Bluebird(async (resolve) => {
 			let countLogs = 0;
-			loki.subscribe(ctx, () => {
+			await loki.subscribe(ctx, () => {
```

Comment:
I feel like this might affect the timings of things

Reply:
I though that given that we already have a `setTimeout(100)` right below to "wait for a bit" it would be fine, but I'm fine keeping it as is for this PR :+1: